docs/deploy-rbd.md

```diff
@@ -280,6 +280,34 @@ Configure a role(s) for service accounts used for ceph-csi:
 * nodeplugin service account (`rbd-csi-nodeplugin`) requires **create** and
   **read** permissions to save new keys and retrieve existing
 
+#### Configuring Amazon KMS
+
+Amazon KMS can be used to encrypt and decrypt the passphrases that are used for
+encrypted RBD images. When a volume is created, a passphrase will be generated,
+which will be encrypted by the KMS and stored in the volumes metadata. Opon
+attaching the volume to a Pod, the worker node requests the KMS to decrypt the
+passphrase, after which it can be used to open the device with `cryptsetup` and
+provide access to it for the Pod.
+
+There are a few settings that need to be included in the [KMS configuration
+file](../examples/kms/vault/kms-config.yaml):
+
+1. `KMS_PROVIDER`: should be set to `aws-metadata`.
```

Comment:
do we need `aws-metadata` or just `aws` enough?

Reply:
At one point we might want to add a KMS implementation that does not store the DEK in the metadata of the volume. Just `aws` would be more suitable for that.

Reply:
Sounds good!